test/index.js

```diff
@@ -26,8 +26,15 @@ fixtures.invalid.forEach(function (f) {
   tape('Invalid for ' + f.description, function (t) {
     t.plan(1)
 
-    var buffer = new Buffer(f.hex, 'hex')
-    var n = pushdata.decode(buffer, 0)
+    var buffer = Buffer.from(f.hex, 'hex')
+    var n
+    try {
+      n = pushdata.decode(buffer, 0)
+    } catch (err) {
+      if (f.hex === '4fffffffff' && err.message === 'Unexpected opcode') {
```

Comment:
Not the best choice... but rewriting the whole fixture structure just to add throw check...

Maybe I'll just make it into a separate tape call at the bottom hard coded into test/index.js...